src/core/layout/qgslayouttable.h

```diff
@@ -461,6 +461,37 @@ class CORE_EXPORT QgsLayoutTable: public QgsLayoutMultiFrame
      */
     QColor backgroundColor() const { return mBackgroundColor; }
 
+    /**
+     * Sets the number of rows before alternating to other color.
+     * \see setBackgroundColor()
+     * \see gridColor()
+     */
+    void setNumberAlternatingRows( int rows );
```

Comment:
Please add \since QGIS 3.32 to all new methods.

Reply:
I have done this, and have also added additional documentation to the commit detailing the new features.